src/WebForms/UI/Page.cs

```diff
@@ -2107,6 +2113,19 @@ internal NameValueCollection GetCollectionBasedOnMethod(bool dontReturnNull)
 #endif
     }
 
+    private bool DetermineIsExportingWebPart()
+    {
+        if (Request.AsAspNetCore().Query.TryGetValue("__WEBPARTEXPORT", out var value) && value is [{ } s] && bool.TryParse(s, out var webPartExport))
```

Comment:
I get confused with these shortcuts. :) .. is this a good programming style (for readability) ? Secondly, in the reference it's using QueryString... should we have a way to expose QueryString (not related to this issue)

Reply:
I'll rework it. I tend to prefer pattern matching but sometimes it can get a bit obtuse 😂

Re query string, you can get that via the core representation, but the original implementation was a weird optimization so I reworked it to be more readable